02-write-your-first-program/exercises/01-print-names/main.go

```diff
@@ -24,6 +25,6 @@ package main
 // ---------------------------------------------------------
 
 func main() {
-	// ?
-	// ?
+	
+	fmt.Println("Hello Shanmuga Rajan Chelliah!@@!!!")
```

Comment:
Hi @shanmugarajan-93,
Those exercises should be in your own git-fork, not here.
I'm closing this issue.